src/benchmarks/micro/libraries/System.Buffers/Base64Tests.cs

```diff
@@ -52,22 +52,22 @@ public void SetupConvertToBase64CharArray()
         [GlobalSetup(Target = nameof(Base64Decode))]
         public void SetupBase64Decode()
         {
-            _encodedBytes = ValuesGenerator.Array<byte>(NumberOfBytes);
```

Comment:
Does it make sense to also keep the ones that operates on invalid data? Just to reflect real world scenario?

Reply:
@DrewScoggins  should be able to confirm

Reply:
We don't write perf tests for failure cases, because we assume they're rare, and often times just bugs that need to be fixed. This is noted in the guidelines -- https://github.com/dotnet/performance/blob/f6066b426c20a5519254b61f9b60153757dc600f/docs/microbenchmark-design-guidelines.md#L288

Reply:
This isn't going to throw an exception. But, yes, all this is testing is "a small number of vector loops and then fallback to scalar"

I'll remove the failure case I added (unless anyone objects)